server/src/test/java/org/elasticsearch/index/query/MatchPhrasePrefixQueryBuilderTests.java

```diff
@@ -39,6 +40,7 @@
 import static org.hamcrest.Matchers.containsString;
 import static org.hamcrest.Matchers.notNullValue;
 
+@LuceneTestCase.AwaitsFix(bugUrl = "should-be-fixed-by-upstream")
```

Comment:
can you elaborate what caused this?

Reply:
These tests are failing on the master branch.